guides/v2.1/cloud/trouble/troubleshoot-deployment.md

```diff
@@ -32,22 +32,25 @@ To redeploy with the correct authentication keys:
 4.	Change to the project root directory.
 5.	Open `auth.json` in a text editor.
 
-    {
-       "http-basic": {
-          "repo.magento.com": {
-             "username": "<your public key>",
-             "password": "<your private key>"
-          }
-       }
-    }
+```json
```

Comment:
This code block must be indented 4 spaces.

Reply:
@jeff-matthews  Added 4 spaces